src/ui/components/pages/MyServices/MyServicesCards/MyServicesCards.tsx

```diff
@@ -71,6 +81,10 @@ export const MyServicesCards = memo((props: Props) => {
         onRequestDelete({ serviceId })
     );
 
+    const onRequestUpgradeFactory = useCallbackFactory(([params]: [UpgradeParams]) =>
```

Comment:
The memoization isn't working here. 
The `params` object ref changes every render.  

```ts
const onRequestUpgradeFactory = useCallbackFactory(([catalogIdd, packageName, name]: [string | undefined, string, string])=> 
  onRequestUpgrade({ catalogId, packageName, name })
);
```

Reply:
Changed after move the upgradeService in core function. It remains just name.